### ISSUE TITLE
About the version of the environment

### ISSUE THREAD

**Issue by PlayerDza** [Clyde013/Paraphrase-OPT#2]
Hello! I am very interested in this project and would like to follow it. But it does not have Instructions for the environment version, such as a file like 'requirements.txt', which confused me a lot, Could you please offer us a requirements file so that we can use the right version of the modules. Thanks!

**Comment by Clydingus** [comment]
Hi @PlayerDza, thanks for your interest, I've committed a requirements.txt file to the repo.
As you're probably aware, simply run `conda create --name <env> --file requirements.txt` to install all the packages.

I cannot in good conscience recommend that anyone actually uses any code I write (although you are of course free to do so if you wish), so if you're interested in similar work, I highly recommend you check out the PEFT repo (https://github.com/huggingface/peft) from HuggingFace that was released a while after work on this repo was completed. It's consistently well-maintained (unlike this one) and contains much better implementations of multiple different fine-tuning techniques, including the prompt fine-tuning implemented here.

Thanks. Hope it works.
